ansible-role/makefile

```diff
@@ -0,0 +1,14 @@
+# Run Molecule tests
+test:
+	molecule test
+
+# Build and test the role
+build: 
+	molecule converge
```

Comment:
Maintain parity b/w the Make target and the command. E.g.:
```
converge:
    molecule converge

# OR

build:
    molecule build
```

Reply:
Done